### PR TITLE
v1.0.1

### PR DESCRIPTION
**Pull request by qsoftdevelopment**
v1.0.1

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -25,20 +28,33 @@ dependencies {
     implementation group: 'com.squareup.retrofit2', name: 'converter-gson', version: '2.6.2'
 
     implementation group: 'org.slf4j', name: 'slf4j-api', version: '1.7.28'
-    testImplementation group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.28'
+    // testImplementation group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.28'
```
Comment (davidnub) [inline build.gradle:31]: Why commented out?

**Comment by parfeon** [comment]
@qsoftdevelopment we need update following sections in `.pubnub.yml` to reflect Kotlin state of things: 
* `features` 
* `supported-platforms`

Also, what version it should be? `build.gradle` says, that previous was `4.0.0` and this one should be actually `4.0.1`?

**Comment by parfeon** [comment]
cc: @kleewho

**Comment by qsoftdevelopment** [comment]
> @davidnub it required some changes from PubNub Kotlin SDK maintainers.
> @qsoftdevelopment we need update following sections in `.pubnub.yml` to reflect Kotlin state of things:
> 
> * `features`
> * `supported-platforms`
> 
> Also, what version it should be? `build.gradle` says, that previous was `4.0.0` and this one should be actually `4.0.1`?

It should be 1.0.1 actually. Kotlin started from 1.0.1 based on the new versioning rule.

File: build.gradle (hunk 1 above)
Comment (qsoftdevelopment) [reply build.gradle]: At that time it was still a work in progress. It has been resolved with the latest work.

File: .pubnub.yml [hunk 2]
```diff
@@ -0,0 +1,115 @@
+name: kotlin
+version: 1.0.0
```
Comment (samiahmedsiddiqui) [inline .pubnub.yml:2]: Version number should be `1.0.1`.

File: .pubnub.yml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+schema: 1
+scm: github.com/pubnub/kotlin
+files:
+  - build/libs/pubnub-kotlin-1.0.0-all.jar
```
Comment (samiahmedsiddiqui) [inline .pubnub.yml:6]: Please change `build/libs/pubnub-kotlin-1.0.0-all.jar` to `build/libs/pubnub-kotlin-1.0.1-all.jar` to point to the latest jar.

File: .pubnub.yml [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+changelog:
```
Comment (samiahmedsiddiqui) [inline .pubnub.yml:7]: Please add `v1.0.1` changes in the changelog.

File: .pubnub.yml (hunk 2 above)
Comment (qsoftdevelopment) [reply .pubnub.yml:2]: The version is fine. This is not 1.0.1 yet.

File: .pubnub.yml (hunk 3 above)
Comment (qsoftdevelopment) [reply .pubnub.yml:6]: The version is fine. This is not 1.0.1 yet.

File: .pubnub.yml (hunk 4 above)
Comment (qsoftdevelopment) [reply .pubnub.yml:7]: The version is fine. This is not 1.0.1 yet.

**Comment by polarweasel** [comment]
Hey @qsoftdevelopment is this ready to merge now?

**Pull request merged**